erpnext/patches/v13_0/add_default_project_into_project_details_child_table.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import unicode_literals
+import frappe
+
+def execute():
+	frappe.reload_doc('projects', 'doctype', 'task')
+	frappe.reload_doc('projects', 'doctype', 'task_project')
```

Comment:
```suggestion
	frappe.reload_doc('projects', 'doctype', 'task')
	frappe.reload_doc('projects', 'doctype', 'task_project')
```
- Reload the task project first and then reload task doctype
- Else the system will throw invalid doctype error